### ISSUE TITLE
Hide/lock survey points

### ISSUE THREAD

**Issue by Dimitar5555** [openstreetmap/iD#9867]
### Description

I've seen cases where mappers liked a survey point and decided to use it for some other object (like a church)... It would be nice if these points are always hidden (with no option to see them) or if their location and tags were immutable in iD.

**Comment by lefuturiste** [comment]
Yeah your probably right, but it's probably better to show them and lock the position. We need to have a transparent experience.

**Comment by RAytoun** [comment]
I am aware of a number of cases where the top of the church spire has been used as a survey point by local land surveyors.

I am not saying that the example you have mentioned is or is not a survey point, but that this should be verified and not just ruled out.

I am 100% in favour of having known fixed anchor points on the map as references for alignment.

**Comment by 1ec5** [comment]
Duplicate of #9255?

**Comment by Dimitar5555** [comment]
> Duplicate of #9255?

#9255 is about survey points that have the same (or very close) coordinates but with diffrence in elevation. This issue is for all survey points.
